config/routes.rb

```diff
@@ -1,7 +1,14 @@
 Rails.application.routes.draw do
   devise_for :users, sign_out_via: [:get, :post]
 
-  
+  namespace :api do
+    resources :users do
+      resources :posts do
+      resources :comments
```

Comment:
- [ ] The API endpoint for creating a post's comment `/api/users/:user_id/posts/:post_id/comments` also throws an error. 

Screenshot 1 |
:-----------------:|
<img width="1122" alt="Screen Shot 2022-11-12 at 5 27 16 PM" src="https://user-images.githubusercontent.com/39523312/201486974-7199219d-73df-41e0-b4ae-a8b74e5d6f01.png"> |

Screenshot 2 |
:----------------:|
<img width="1132" alt="Screen Shot 2022-11-12 at 5 27 31 PM" src="https://user-images.githubusercontent.com/39523312/201486983-7ffbf246-c403-45f4-b1c4-7a75c5db02a1.png"> |

I want to suggest please take a look at them, verify it works well on your end and ask for a re-review.